.github/workflows/validate_datasets.yml

```diff
@@ -65,7 +100,7 @@ jobs:
           deno install -Agf -n bids-validator jsr:@bids/validator
 
     - name: Install BIDS validator (main)
-      if: matrix.bids-validator == 'main'
+      if: matrix.bids-validator == 'main' || matrix.bids-validator == 'bids-pr'
```

Comment:
In general, `dev` tracks unreleased changes to the schema, while `main` tracks the latest released version of BIDS. PRs against the spec will mostly aim for `dev`.

Reply:
yes, but it is useful to know if it fails against released version since if it doesn't but expects to (e.g. new file type was introduced like `atlas-<label>_description.json` but released validator+schema is ok with that means that either they miss something or something else is not stringent enough.  The same about `dev` as -- have we made development validator and/or schema less stringent, as we would expect then only the `dev-pr` to pass in such cases.   That is why I preferred to keep it separate and additional matrix run.